content/docs/01-introduction.md

```diff
@@ -1,18 +1,71 @@
 
+Palette offers the most comprehensive profile-based management for Kubernetes. It enables teams to drive consistency, repeatability, and operational efficiency across multiple clusters in multiple environments with comprehensive day 0 - day 2 management.
+ 
+### Any Environment
 
-# Welcome to **Spectro Cloud**
+Palette has the richest coverage in supported environments that includes:
+- Public Clouds: AWS, Azure, and Google Cloud (both IaaS and managed Kubernetes services EKS/AKS/GKE)
+- Data Centers: VMware, OpenStack
+- Bare Metal: Canonical MaaS
+- Edge
+ 
+ 
+ 
+## What is Under the Hood?
 
-Welcome to Spectro Cloud. We aim to make Kubernetes accessible and manageable with a simple to use platform to manage any combination of Kubernetes clusters across any environment.
\ No newline at end of file
+Palette uniquely extends and integrates the Cloud Native Computing Foundation (CNCF) open-source Cluster API project. Palette does this by providing comprehensive full-stack modeling and orchestration, governance, security, and day 0 - day 2 management capabilities.
+ 
+With Palette’s Cluster Profiles, teams can define full-stack clusters that include both the Kubernetes infrastructure and any add-on application services. Cluster Profiles enable a repeatable way to deploy and reuse clusters across any environment. Palette also enables importing of existing Kubernetes environments and creating equivalent Cluster Profiles.
+ 
+ 
+![2-what-is-sc](/docs_introduction_palette-components.png)
+ 
+## Who Can Benefit From Palette?
+ 
+### Developers
+
+Development teams will get the flexibility and freedom they are looking for to increase the speed of innovation, whether it is the cluster template with the add-on application services or choosing a Kubernetes version with integrations like logging, monitoring, and service mesh for your application development. They need not worry about Kubernetes configurations but focus on the stuff that matters.
+ 
+### IT Operations and SREs
+ 
+Declarative management makes life easier for IT teams, with consistency, repeatability, and all the enterprise-grade controls and governance they need - especially when moving to production [Cluster Profiles](/glossary-all#clusterprofile) enable them to define and re-use full-stack clusters and support them across the entire lifecycle without having to write scripts, as well as integrate with existing tools and methodologies.
+ 
+ 
+### IT Executives
+ 
+With an open and enterprise-grade platform, IT leaders can get peace of mind without being locked into proprietary orchestration technologies or one-size-fits-all solutions. This helps lower the total cost of ownership (TCO) and reduce operational risk.
+
+
+# Next Steps
+Learn more about Palette and how it can improve your Kubernete experience but also for your organization. Try [Palette](https://console.spectrocloud.com/) for free today and experience Kubernetes unlike anything before. 
```

Comment:
```suggestion
Learn more about Palette and how it can improve your Kubernetes experience and those in your organization. Try [Palette](https://console.spectrocloud.com/) for free today and experience a better way of working with Kubernetes. 
```